distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -937,13 +951,25 @@ type Table
                  table = Examples.inventory_table
                  double_inventory = table.at "total_stock" * 2
                  table.set "total_stock" double_inventory
-    set : Text -> Column | Vector.Vector -> Table
-    set self name column = case column of
-        _ : Vector.Vector ->
-            self.set name (Column.from_vector name column)
+    set : Text -> Column | Vector.Vector | Text -> Problem_Behavior -> Table
+    set self name column on_problems=Report_Warning = case column of
+        _ : Text ->
+            expression = self.evaluate column
+            if expression.is_error.not then self.set name expression on_problems else
+                on_problems.handle_errors expression fallback=self
+        _ : Vector.Vector -> self.set name (Column.from_vector name column)
         Column.Column_Data _ ->
```

Comment:
```suggestion
        _ : Column ->
```

nitpick as both forms are ok, but since we have the new nice type-match syntax, I'd prefer to use it where possible - it is more robust as does not depend on changes of constructor name/arity